src/tools/_tools.ms.scss

```diff
@@ -0,0 +1,9 @@
+// /*------------------------------------*\
+//     #MS
+// \*------------------------------------*/
+
+// To clearly indicate that a scale value is used, use it with a custom
```

Comment:
I don't quite get the purpose of this function if it outputs the input value

Reply:
It really is just to clearly indicate that a modular scale value is used, e.g. in module variables:

```scss
$foo-font-size: ms(3) !default;

.foo {
    @include fluidms-font-size(
        $font-size: $foo-font-size
    );
}
```

This could be indeed also just be written like this with the same resulting output:

```scss
$foo-font-size: 3 !default;

.foo {
    @include fluidms-font-size(
        $font-size: $foo-font-size
    );
}
```

However with the latter example, it isn’t instantly clear, that this is a modular scale entity — it could just as well be a CSS length value with a missing unit, e.g. `3` (with missing `em`).

The whole calculation cannot be moved to the `ms()` function so the function is really just for this cosmetic purpose.

Reply:
Got it.
Yeah, I agree that `$module-variable: ms(x)` is way more clear than `$module-variable: x`